code/modules/spells/aoe_turf/conjure/forcewall.dm

```diff
@@ -69,3 +104,20 @@
 	new /obj/effect/forcefield/cult(get_turf(src))
 	qdel(src)
 	return
+
+/obj/effect/unwall_field
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "fuel"
+	name = "invisible un-wall"
+	desc = "You have a REALLY bad feeling about this."
+	anchored = 1.0
+	opacity = 0
+	var/static/list/forbidden_passes = list(/turf/unsimulated/wall,/turf/simulated/wall/invulnerable,/obj/structure/grille/invulnerable)
+
+/obj/effect/unwall_field/to_bump(atom/A)
+	if(ismovable(A))
+		var/atom/movable/AM = A
+		for(var/atom/B in src.loc)
```

Comment:
Two of the forbidden types are turfs themselves, which would not be in their own `contents`.

Reply:
Fixed